topologic/embedding/omnibus_embedding.py

```diff
@@ -92,22 +93,37 @@ def omnibus_embedding(
 
     embedding_containers = []
 
-    previous_graph = largest_connected_component(graphs[0])
+    # union graph lcc strategy
+    union_graph = graphs[0].copy()
+    for graph in graphs[1:]:
+        for s, t, w in graph.edges(data='weight'):
```

Comment:
can you use full words here, not just single letters?  `source, target, weight`?